db/seeds/fincap.seeds.rb

```diff
@@ -481,6 +493,8 @@
 tag = Tag.create(
   value: 'how-can-we-improve-the-financial-capability-of-young-adults'
 )
+mobile_payments_tag = Tag.create(value: 'mobile-payments')
+secure_payments_tag = Tag.create(value: 'secure-payments')
```

Comment:
Is there a reason for the tags?

Reply:
@tomas-stefano  To ensure they appeared on the page. I will also need them for the Latest News Component. A bit premature and I can remove them if you want